sdcm/cluster.py

```diff
@@ -4419,6 +4419,8 @@ def node_setup(self, node: BaseNode, verbose: bool = False, timeout: int = 3600)
 
         if not self.test_config.REUSE_CLUSTER:
             node.disable_daily_triggered_services()
+            SyslogNgExporterSetup().install(node)
```

Comment:
shouldn't we fence it with `if self.params.get('logs_transport') == 'syslog-ng':` so we don't install it for e.g. artifact tests where we have `ssh` backend?

Reply:
yeah, we probably should

Reply:
fixed